tests/benchmarks/exercism_practice.py

```diff
@@ -133,33 +98,40 @@ def language(request):
     return request.config.getoption("--language")
 
 
-prompt = dedent("""\
-    You are a professional code reviewer who helps other coders improve their skills.
-    You recently assigned a coder a small coding test to assess their level, with a pre-written stub template
-    and an automated test suite to determine if they succeeded.
-    They failed the test; using the instructions for the test,
-    the code they wrote for the test, and the output of the test suite,
-    your job is to determine why they failed. Give a terse but informative couple of sentences
-    on why it failed, before giving the reason it failed on the final line in the format
-    reason: <reason_failed>
-    Your response will be parsed programmatically, so you MUST follow the format for the final line!
-    The possible responses for the final line and what they mean are as follows:
-    blank (the coder didn't change the file at all from the stub you provided them)
-    wording (everything was correct, but the coder messed up the wording or spacing which caused it to be rejected)
-    duplication (the coder had a random duplicated line that caused the code to not be compiled/interpreted)
-    syntax (the coder messed up their syntax, meaning their code couldn't be compiled/interpreted)
-    logic (the coder messed up the logic)
-    other (some other reason caused it to fail)""")
+prompt = (
+    "You are a professional code reviewer who helps other coders improve their skills."
```

Comment:
Hmm, I think we still want newlines and spaces in a lot of places here; I'm going to fix this up a bit before merging